R/track-util.R

```diff
@@ -0,0 +1,142 @@
+#' track helpers
```

Comment:
In this situation, I think it’s indicative of too much decoupling. 

Each function is called exactly once by the same function in the same place (`cytominer::track`). I’d recommend moving these functions into track.R.